lib/fizzy/parser_expr.cpp

```diff
@@ -209,6 +209,9 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end)
 
         case Instr::block:
         {
+            if (pos == end)
```

Comment:
Why not move these to line 43?

Reply:
Only having it in cases where there's an immediate?

Actually, why are we not putting arity onto the immediate list?